README.md

```diff
@@ -1,5 +1,40 @@
 # iRODS C++ REST Mid-Tier API
 
+This REST API is designed to be run alongside an iRODS Server to provide an HTTP REST interface to the majority of administrative functions for an iRODS Zone.
+
+It will run under the same linux service account and access `/etc/irods/server_config.json` and the active authenticated `rodsadmin` iRODS account.
+
+This REST API requires an iRODS Server v4.2.5 or greater.
```

Comment:
4.2.5 may be too early.

This project uses (without scanning every line of code):
- query iterator
- dstream
- filesystem
- resource administration
- at_scope_exit

There might be other libs it uses as well. I'll check and report back here.

Reply:
Ah, right - the limiting factor is the 'latest' of the used libraries, not the oldest.

Reply:
Going with 4.2.0 for now...

This is not about the implementations within each of irods_client_rest_cpp or the server itself... it's about iRODS protocol compatibility and, I think... the `resc_name` to `resc_id` conversion in 4.2.0.